docs/bsp.md

```diff
@@ -377,16 +380,21 @@ object MessageType {
 }
 ```
 
+A `build/showMessage` notification is similar to LSP's `window/showMessage`, except for a few new
+additions.
+
+The `id` and optional `parentId` fields allow clients to structure logs in a hierarchical way (in a
+tree fashion, with dropdowns, et cetera) to ease readability.
```

Comment:
s/, et cetera//

Same goes for "obviously" and "clearly", makes readers feel dumb when they don't understand